gordo/builder/build_model.py

```diff
@@ -212,8 +212,20 @@ def _build(self) -> Tuple[sklearn.base.BaseEstimator, Machine]:
                 scaler = self.machine.evaluation.get("scoring_scaler")
                 metrics_dict = self.build_metrics_dict(metrics_list, y, scaler=scaler)
 
+                try:
+                    n_splits = int(
+                        self.machine.evaluation["cv"][
+                            "sklearn.model_selection.TimeSeriesSplit"
+                        ]["n_splits"]
+                    )
```

Comment:
Think we should might as well allow complete parameterization here with something like:
```python
serializer.pipeline_from_definition(self.machine.evalutation.get("cv", some_default))
```

Reply:
I see, I will fix this.